src/libs/Url.js

```diff
@@ -38,26 +38,38 @@ function getURLObject(href) {
     };
 }
 
+/**
+ * Determine if we should remove w3 from hostname
+ * @param {String} hostname
+ * @returns {Boolean}
+ */
+function shouldRemoveW3(hostname) {
+    const parts = hostname.split('.').reverse();
+    const subDomain = parts[2];
+    return subDomain === 'www';
+}
```

Comment:
It is good to create a separate function for this but there is not base for the logic used in this. why are we checking parts[2] for subdomain? URL can be very versatile. That's why I suggest you just move this logic inside `hasSameExpensifyOrigin` so that it is clear that `subDomain === 'www'` is applicable to Expensify domains.

Reply:
We have www.new.expensify.com => hostname.split('.').reverse() = [com,expensify,new,www] that why I use parts[2] (new) for subdomain. I'm also afraid of duplicate this logic inside `hasSameExpensifyOrigin` so I created a separate function.

Reply:
Can we use shouldRemoveW3FromExpensifyUrl?

Reply:
Yeah, I didn't notice there are two calls to it. Yeah that will work.

Reply:
@parasharrajat I've change my PR. Pls help check, thanks